src/components/hand-controls.js

```diff
@@ -156,25 +156,31 @@ module.exports.Component = registerComponent('hand-controls', {
     return controllerId === 'OpenVR Gamepad';
   },
 
+  isOculusTouchController: function () {
```

Comment:
Is `isViveController` method used? Can we remove? Should we move these methods to utils?